java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/SparkPlanner.scala

```diff
@@ -64,11 +68,19 @@ class SparkPlanner(session: SparkSession, config: OpenmldbBatchConfig, sparkAppN
     this(session, OpenmldbBatchConfig.fromSparkSession(session), session.conf.get("spark.app.name"))
   }
 
+  def this(openmldbSession: OpenmldbSession, config: OpenmldbBatchConfig) = {
```

Comment:
We may put `openmldbSession: OpenmldbSession` in the `class SparkPlanner(session: SparkSession, config: OpenmldbBatchConfig, sparkAppName: String)` which requires users to set.